src/sandbox/cgroupFactory.go

```diff
@@ -29,6 +29,7 @@ type Cgroup struct {
 
 type CgroupPool struct {
 	Name     string
+	memPool  *MemPool
```

Comment:
I don't think the MemPool abstraction should bleed across layers.  The cgroup pool should be dumb (i.e., it doesn't do policy), and should just expose an API like this: changeMemLimit(mb int)

The sock.go layer should be the one to decide when to upsize/downsize the memory limit.  I was suggesting changing the Pause/Unpause functions in sock.go, rather than implement that logic here in cgroupFactory.go.

One reason for implementing the mechanism, changeMemLimit(mb int), here and the resizing policy in sock.go is that there are other opportunities to play with the mem limits that we can only do at a higher level.  For example, fork servers are unlikely to grow in memory after they've done they're imports, so we should eventually consider constricting them (paused or not).